app/javascript/chat/chat.jsx

```diff
@@ -664,10 +664,11 @@ export class Chat extends Component {
       });
     }
     if (messageIsEmpty) {
-      const messagesByCurrentUser = messages[activeChannelId].filter(
+      const messagesByCurrentUser = messages[activeChannelId]?.filter(
```

Comment:
The changes in this file aren't really related to the bugfix at hand, but it was causing some unhandled exceptions that made the cypress test fail 🙃 

I don't really like this whole chunk of code but I'm conscious of scope creep and refactoring it doesn't really belong in this PR. The remainder of changes in this file are just formatting via prettier